src/app/core/user/user-list/user-list.component.ts

```diff
@@ -1,9 +1,99 @@
-import { Component } from "@angular/core";
+import { Component, inject, signal, OnInit } from "@angular/core";
+import { UserAdminService } from "../user-admin-service/user-admin.service";
+import { MatDialog } from "@angular/material/dialog";
+import { SessionSubject } from "../../session/auth/session-info";
+import { UserAccount } from "../user-admin-service/user-account";
+import { Logging } from "../../logging/logging.service";
+import {
+  UserDetailsAction,
+  UserDetailsComponent,
+  UserDetailsDialogData,
+} from "../user-details/user-details.component";
+import { ViewTitleComponent } from "../../common-components/view-title/view-title.component";
+import { MatTableModule } from "@angular/material/table";
+import { EntityBlockComponent } from "../../basic-datatypes/entity/entity-block/entity-block.component";
 
 @Component({
   selector: "app-user-list",
-  imports: [],
+  imports: [ViewTitleComponent, MatTableModule, EntityBlockComponent],
+
   templateUrl: "./user-list.component.html",
   styleUrl: "./user-list.component.scss",
 })
-export class UserListComponent {}
+export class UserListComponent implements OnInit {
+  private readonly userAdminService = inject(UserAdminService);
+  private readonly dialog = inject(MatDialog);
+  private readonly sessionInfo = inject(SessionSubject);
+
+  users = signal<UserAccount[]>([]);
+  displayedColumns: string[] = [
+    "email",
+    "userEntityId",
+    "enabled",
+    "emailVerified",
+    "roles",
+  ];
+
+  ngOnInit() {
+    this.loadUsers();
```

Comment:
Let's check permissions here before attempting to load (or do that within userAdminService). We can avoid throwing (and logging) errors for
- anyone who does not have access, missing the account_manager role
- situations where the device is offline currently